app/representers/api/auth/feed_representer.rb

```diff
@@ -17,11 +17,21 @@ class Api::Auth::FeedRepresenter < Api::BaseRepresenter
   property :include_tags
   property :audio_format
 
+  # NOTE: bypass representers here, so getting/setting works correctly
+  property :represented_tokens, as: :tokens
+
   def self_url(feed)    
     api_authorization_podcast_feed_path(podcast_id: feed.podcast_id, id: feed.id)
   end
 
   link :podcast do
     api_authorization_podcast_path(represented.podcast) if represented.id && represented.podcast
   end
+
+  link :rss do
+    {
+      href: represented.published_url,
+      templated: true
+    }
+  end
```

Comment:
I'm a bit lukewarm on this one.

The Podcast representer has a non-writeable _property_ called `published_url`.  Which is the default feed's `published_url`.  Not sure why that isn't a `_link`?

But here, especially because the value could be templated (private feeds need to include the `?auth=1234` token) - I made it a link.  Bad idea?  Maybe.

I'm also lukewarm on what to call the link.  Went with `prx:rss` here.

Reply:
The Podcast representer having a url as a read only property instead of a link is an error.

I would compare this with how the episode representer has a feed _link, and an enclosure _link, which was a better way to do it, and that's consistent with this addition of a private-feed _link

in other words, this looks right to me, and Podcast representer published_url looks wrong